bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/DYBJets_HT-incl_Zpt-100to200/DYBJets_HT-incl_Zpt-100to200_proc_card.dat

```diff
@@ -0,0 +1,21 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_color_flows False
+set gauge unitary
+set complex_mass_scheme False
+set max_npoint_for_channel 0
+import model sm-ckm_no_b_mass
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+define p = p b b~
+define j = p
+define bquark = b b~
+define ell+ = e+ mu+ ta+
+define ell- = e- mu- ta-
```

Comment:
please remove everything that is not relevant for you. defintions/models you don't use, default mg5 settings ...